src/pytkdocs/loader.py

```diff
@@ -351,6 +353,23 @@ def get_class_documentation(self, node: ObjectNode, select_members=None) -> Clas
 
         select_members = select_members or set()
 
+        # Blacklist inherited marshmallow properties
+        blacklist = []
+        if issubclass(class_, Schema):
+            blacklist += [
+                "__class__",
+                "Meta",
+                "OPTIONS_CLASS",
+                "dump",
+                "dumps",
+                "get_attribute",
+                "handle_error",
+                "load",
+                "loads",
+                "on_bind_field",
+                "validate",
+            ]
+
```

Comment:
pytkdocs should never hardcode-exclude anything. Some users might want to get the data for these exact attributes that are you are excluding. Instead, you can filter them out with the `filters` and `members` options, which are available in mkdocstrings as well.

Reply:
It would be nice to be able to preset these then, i dont want to have to add filter [10 items] for every ::: marshmallow.Schema in my docs (i use schemas extensively in the app i am working on). 

Can i set a default filter for this type of object maybe?

Reply:
Filters for specific types of objects are not possible as of now, but you can set global filters in `mkdocs.yml`: https://pawamoy.github.io/mkdocstrings/handlers/python/#selection

Feel free to open a new issue for discussing better ways of filtering / selecting objects in pytkdocs or mkdocstrings :slightly_smiling_face: